README.md

```diff
@@ -269,11 +362,15 @@ While we each bring a different perspective and set of expertise to customer int
 
 # Company policies: our expectations of each other
 
+## Information security and privacy :lock:
+
+Able Health's customers entrust us with a significant amount of protected health information (PHI). As such, we take great care to ensure the security and privacy of potentially sensitive information. All team members receive HIPAA training and may refer to our HIPAA policies and procedures at any time for further details.
+
 ## Team meetings :clipboard:
 
-* Product standup at 9:50am every day
+* Product standup at 9:45am every day
```

Comment:
Well, when I created the separate standup it was the "engineering huddle" from Day 1. It's really an engineering check-in, in the context of the backlog. So calling it "Product standup" isn't what I would choose.